common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3147,12 +3147,11 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
       new RangeValidator(0, 100), "Determines how many failed compaction records will be " +
       "retained in compaction history for a given table/partition."),
     /**
-     * @deprecated Use MetastoreConf.COMPACTOR_HISTORY_RETENTION_ATTEMPTED
+     * @deprecated Use MetastoreConf.COMPACTOR_HISTORY_RETENTION_DID_NOT_INITIATE
      */
-    @Deprecated
-    COMPACTOR_HISTORY_RETENTION_ATTEMPTED("hive.compactor.history.retention.attempted", 2,
-      new RangeValidator(0, 100), "Determines how many attempted compaction records will be " +
-      "retained in compaction history for a given table/partition."),
+    @Deprecated COMPACTOR_HISTORY_RETENTION_DID_NOT_INITIATE("hive.compactor.history.retention.attempted", 2,
```

Comment:
Could we use configuration deprecation to change the name of the config as well? Otherwise we will carry this baggage forever.